components/client/src/test/unit/resources/logback.xml

```diff
@@ -16,6 +16,5 @@
   <logger name="com.hotels.styx.http-messages.inbound" level="INFO"/>
   <logger name="com.hotels.styx.http-messages.outbound" level="INFO"/>
   <logger name="com.hotels.styx.client.OriginRestrictionLoadBalancingStrategy" level="INFO"/>
-  <logger name="com.hotels.styx.client.netty.connectionpool.FlowControllingHttpContentProducer" level="INFO"/>
```

Comment:
should we remove this or just rename with the appropriate package name?

Reply:
remove it as the class has moved to a different module